client/package.json

```diff
@@ -137,6 +137,7 @@
 		"immutability-helper": "^3.0.1",
 		"inherits": "^2.0.4",
 		"is-my-json-valid": "^2.20.5",
+		"is-plain-obj": "^3.0.0",
```

Comment:
Why v3 rather than v4?

Reply:
From 4.0 it's pure ESM and that will break both our server build and our tests. Currently, we don't transpile node modules for a good reason, so to fix that we might have to transpile this module.